File: src/main/kotlin/wowsft/model/gameparams/ship/abilities/AbilitySlot.java
package wowsft.model.gameparams.ship.abilities;

import wowsft.config.WoWSFT;
import com.fasterxml.jackson.annotation.JsonInclude;

import java.util.List;

@WoWSFT
public class AbilitySlot
{
    private List<List<String>> abils;
    @JsonInclude
    private int slot;
}

File: src/main/kotlin/wowsft/model/gameparams/ship/abilities/AbilitySlot.kt
package wowsft.model.gameparams.ship.abilities

import wowsft.config.WoWSFT
import com.fasterxml.jackson.annotation.JsonInclude

@WoWSFT
class AbilitySlot {
    var abils = ArrayList<MutableList<String>>()
    @JsonInclude
    var slot = 0
}
